### DISCUSSION TITLE
Center Search Input

### DISCUSSION THREAD

**Discussion by rwxsb** [community/community#58561] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I see that with new UI search input is aligned to right side
![image](https://github.com/community/community/assets/20385029/6d9a05b3-d9dc-4d36-a642-d4fe89188c57)
I suggest keeping it on the left side, where we're accustomed to seeing it, or at least moving it to the center. This approach will make it easier, as we tend to look in these areas first due to habit.

**Comment by GhostOf0days** [comment]
Fun Fact: It's not because you're used to it. Users naturally look at any website/layout in F patterns and Z patterns. There's even academic research on it. Here's an [article](https://lineindesign.medium.com/be-a-designer-who-can-also-help-with-writing-copy-2f4ea02a5646) on it!
